packages/router-plugin/tests/router.plugin.spec.ts

```diff
@@ -103,18 +126,17 @@ describe('NgxsRouterPlugin', () => {
 
     const store: Store = TestBed.get(Store);
 
-    store.dispatch(new Navigate(['a-path'], { foo: 'bar' }));
-    tick();
+    await store.dispatch(new Navigate(['a-path'], { foo: 'bar' })).toPromise();
```

Comment:
why await instead tick?

Reply:
I think code that relies on js behavior is better than code that relies in zone.js
This makes the test more reliable IMO

Reply:
Well, then it will be necessary to rewrite a lot of code anyway from zone.js?)